.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -4,7 +4,17 @@ on:
     branches:
       - main
       - prerelease_test
+    paths-ignore:
+      - '**.md'
+      - LICENSE
+      - NOTICE
+      - 'bootstrap*'
```

Comment:
Why are the bootstrap files excluded? Shouldn't we test the commit when they change?

Reply:
We are not using the bootstrap file for windows, but we are using it for the linux builds. Fixed.